modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/CliRequests.java

```diff
@@ -89,7 +99,8 @@ public interface ChangePeersAsyncRequest extends Message {
 
         Collection<String> newLearnersList();
 
-        long term();
+        // term is intentionally Long and not long in order to perform nullable (not initialized) check.
```

Comment:
I'm sorry, didn't understand - it is not mandatory (not @Nullable), but can be null at the same time?

Reply:
It's mandatory because it's not marked as @Nullable. If someone will forget to specify term() while building the request AssertionError will be thrown, something like "request field not marked with @nullable has null value".